pkg/operator/register.go

```diff
@@ -49,11 +49,11 @@ func RegisterCmd(p utils.Prompter) *cli.Command {
 				utils.EmojiCheckMark,
 				operatorCfg.Operator.Address,
 			)
-			fmt.Printf("%s validating operator config: %s", utils.EmojiWait, operatorCfg.Operator.Address)
+			fmt.Printf("%s validating operator config: %s ... ", utils.EmojiWait, operatorCfg.Operator.Address)
```

Comment:
@andrewkmin Since we are doing carriage return and overriding the `EmojiWait` - I think ideal fix would be to add `\r` on line `56` - take a look at line `60` in this file. wdyt?

Reply:
Oops, sorry for missing this all this time! Noticed there have been some refactors in the mean time. I like the idea of `\r` -- just updated the PR